neofetch

```diff
@@ -2041,6 +2051,17 @@ get_gpu() {
                "${gpus[1]}" == *Intel* ]] && \
                unset -v "gpus[0]"
 
+            # If nvidia-smi is installed, use that for NVIDIA GPUs
+            nv_smi_cmd="$(command -v nvidia-smi)"
+            if [[ -x "$nv_smi_cmd" ]]; then
+                nv_gpu_cmd="$nv_smi_cmd --query-gpu=name --format=csv,noheader"
+                IFS=$'\n' read -d "" -ra nv_gpus <<< "$($nv_gpu_cmd)"
+                for nv_gpu in "${nv_gpus[@]}"; do
+                    gpu="NVIDIA $nv_gpu"
```

Comment:
The `nvidia-smi` command already has "NVIDIA" in it, so we don't need to add NVIDIA here

![image](https://user-images.githubusercontent.com/22280294/184279280-b7c2b48b-67f9-4910-abed-f07783508861.png)